ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -29,10 +29,11 @@ class WaypointUpdater(object):
     def __init__(self):
         rospy.init_node('waypoint_updater')
 
+        self.speed_limit = rospy.get_param('/waypoint_loader/velocity', 15) * 0.45 # MPH to m/s
```

Comment:
I think waypoint loader velocity param is Kilometer per hour.
40 KPH = 100/9 m/s = 24.85 MPH
The multiplier is then 5/18 = 0.277778

Reply:
yes your right, that makes sense.